src/main/java/com/github/britter/springbootherokudemo/entity/enumeration/RoomStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.github.britter.springbootherokudemo.entity.enumeration;
+
+public enum RoomStatus {
+    AVAILABLE, OCCUPIED, UNKNOWN
```

Comment:
Tak jak napisałeś - to jest tylko zmiana przygotowawcza - zmiany w starym API bez widocznych zmian "z zewnątrz", ale masz racje - lepiej wmegować od razu wszystko, więc jak zrobie to wrzuce PR z dwoma commitami, więc ten można na razie odrzucić